src/ui/marker.js

```diff
@@ -548,6 +562,25 @@ export default class Marker extends Evented {
         return this._draggable;
     }
 
+    /**
+     * Sets the `scale` property of the default marker.
+     * @param {number} [scale=1] The scale of the marker, relative to its default size.
+     * @returns {Marker} `this`
+     */
+    setScale(scale: number) {
+        this._scale = scale || 1;
+        this._update();
+        return this;
+    }
+
+    /**
+     * Returns the current `scale` property of the default marker.
+     * @returns {number} The current scale of the marker.
+     */
+    getScale() {
+        return this._scale;
+    }
+
```

Comment:
So....I know that I suggested these originally, but I misunderstood the PR and now I think it would probably be best to remove these methods. Sorry :(

My reasoning now is that this PR only applies to the default SVG marker. I don't think there's any way to universally resize HTML elements regardless of type, so if a user supplies their own marker element, these methods won't work. Given that, we should avoid adding methods to the API that only work some of the time. It might be best to rely only on the scale option.

Reply:
No worries, that reasoning sounds good to me! I do understand why a developer using these methods with a custom HTML element for their marker would have a strange experience. Even though calling `Marker#setScale` would have no effect if `this._defaultMarker` is `false`, providing methods which have an effect conditioned on the value of a parameter is not ideal.

Reply:
I can see both sides here, one use case for changing the scale is having a marker made bigger when you select it, however I can see how since this only applies to the default marker it would be confusing to have a setScale which does nothing.

To still support this make it larger when selected use case, I do lean on the side that we should support get/setScale, with a note in the JS doc that it only applies when using the default marker, and a console warning if used on a non-default marker?

Reply:
I do also see use cases where the getter + setter for `scale` would be useful, and I'd be open to adding the methods back. It is a bit limiting to only be able to specify marker sizing at creation. I do understand how these methods would be an "exception to the rule," however, since currently there are no other user interface components in the API with similarly "conditional" instance members.

If `Marker#getScale` and `Marker#setScale` were included, it would probably make sense to add `Marker#getColor` and `Marker#setColor` too, since `scale` and `color` are very similar. Though, that is outside the scope of this PR.